script_library/run-ansible.sh

```diff
@@ -29,21 +33,18 @@ function run_ansible(){
     fi
 
     if [[ -f ${extravarsfile} ]]; then
-        echo "Extra variables file exists: $(realpath ${extravarsfile}). Loading its vars in ansible-playbook call."
-        extra_vars="-e @${extravarsfile}"
+        echo "Extra variables file exists: $(realpath "${extravarsfile}"). Loading its vars in ansible-playbook call."
+        ansible_command="$ansible_command -e ${extravarsfile}"
```

Comment:
If there is no extravars file, the ansible_command will not be set, is that right?

Reply:
nope, if there is no extravars then the extravars switch wont be added to the command, the command `ansible_command` is already defined at the top of the file

Reply:
Sorry I didn't see that :p Thanks!